### ISSUE TITLE
New emoji combinations

### ISSUE THREAD

**Issue by cyb3rko** [xsalazar/emoji-kitchen#50]
And once again, there are A LOT of new combinations like 

![markup_14265.png](https://user-images.githubusercontent.com/46033730/160019716-c32ac0f8-4e31-4e8e-aae5-49fd75a1f9e3.png)

![Screenshot_20220324-231759.png](https://user-images.githubusercontent.com/46033730/160019490-52fc1064-d3e2-4dac-9dba-e84a6c4d704d.png)

For the full list of updates see here:
https://9to5google.com/2022/03/24/gboard-emoji-kitchen-list/amp/

**Comment by xsalazar** [comment]
👋 Thanks @cyb3rko! I think I have most of these already live at https://emojikitchen.dev!

The only one I see missing is :tm:, which I can add here soon.

These lists are hard to parse because a _lot_ of these emojis alias to other ones, and I'm only adding the original, source emoji in this project. I usually have been going through manually to prune out the duplicates, but it's likely I may be missing one.

Conceptually, it makes it a bit easier to browse and find 🩸 and their combos, but then trying 🆎 just aliases to all the same emojis so that's not as useful.

It's also worth noting that I chose to do this because I've not run across a combo which is _both_ an alias **and** has a new combo associated with it. Continuing the blood example from above, I don't see _new_ 🆎 combination; rather, it seems to be entirely identical to the 🩸 combinations.

**Comment by xsalazar** [comment]
:tm: Added as part of https://github.com/xsalazar/emoji-kitchen/commit/bafec6f69865bce4c391f5a5efb48932be247190 🚀
